.github/actions/run-tests/action.yml

```diff
@@ -12,7 +12,7 @@ runs:
       run: ${{ inputs.command }}
 
     - name: Upload Test Results
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

Comment:
in fact this action and the relative "publish test results" job in the `verify` have been removed so long ago from all the other repos, they can be removed from here as well. Different issue tho :)